src/main/build/build_config.h

```diff
@@ -56,7 +56,9 @@ typedef enum {
     MCU_TYPE_H743_REV_X,
     MCU_TYPE_H743_REV_V,
     MCU_TYPE_H7A3,
-    MCU_TYPE_H723_725,
+    MCU_TYPE_H723,
+    MCU_TYPE_H725,
```

Comment:
This will change the IDs which will break previous configurators. Need to add new ones to the end. Reuse the existing as the most used eg H723

Reply:
Previous configurator versions do not work with 4.6, but we need to back port and change config, so introducing these changes might be an issue with firmware < 4.5.2